.github/workflows/publish-release.yml

```diff
@@ -0,0 +1,47 @@
+name: Publish release
+
+on:
+  push:
+    tags:
+      - 'v*'
+
+jobs:
+  ig-publisher:
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: nowsprinting/check-version-format-action@v3
+        id: version
+        with:
+          prefix: 'v'
+
+      - name: Version tag only step
+        run: |
+          echo "Tag format is invalid: double check that it is a valid semver tag"
+          exit 1
+        if: steps.version.outputs.is_valid != 'true'
+    
+      - uses: actions/checkout@v3
+
+      - name: Install Jekyll
```

Comment:
I see same steps in 'Validate docs' workflow as well. in gitHub, Can't we use same workspace for all the workflows? then we need not to install again the same.

Reply:
We can, but this is a case of [AHA](https://www.franciscomoretti.com/code-tips/avoid-hasty-abstractions-aha) - the two workflows are for totally different purposes.

Even though the steps look the same, the script would become a mess as should it try to fulfill two different jobs.